app/javascript/react/utils/timeFormatter.ts

```diff
@@ -0,0 +1,5 @@
+import moment from "moment";
+
+export const formattedTime = (time: string, format: string) => {
+  return moment.utc(time).format(format);
```

Comment:
I think this is the time discrepancy culprit.

Reply:
This was also my guess  :)